CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
+    - Note this is only in effect when `using_schedule_histories` is true.
+  - This improves granularity for Zendesk metrics related to agent availability, SLA tracking, and time-based performance analysis.
+### dbt_zendesk_source changes (see the [Release Notes](https://github.com/fivetran/dbt_zendesk_source/releases/tag/v0.13.0) for more details)
+- Introduced the `stg_zendesk__audit_log` table for capturing schedule changes from Zendesk's audit log.
+  - This model is disabled by default, to enable it set variable `using_schedule_histories` to `true` in `dbt_project.yml`.
+
+## New Features
+- Holiday support: Users can now choose to disable holiday tracking by setting variable `using_holidays` to `false` in `dbt_project.yml`.
+- New intermediate models have been introduced to streamline both the readability and maintainability:
+  - `int_zendesk__timezone_daylight`: A utility model that maintains a record of daylight savings adjustments for each time zone.
+    - materialization: ephemeral
+  - `int_zendesk__schedule_history`: Captures a full history of schedule changes for each `schedule_id`.
+    - materialization: table (if enabled)
+  - `int_zendesk__schedule_timezones`: Merges schedule history with time zone shifts.
+    - materialization: ephemeral
+  - `int_zendesk__schedule_holidays`: Identifies and calculates holiday periods for each schedule.
+    - materialization: ephemeral
+- Rebuilt logic in `int_zendesk__schedule_spine` to consolidate updates from the new intermediate models.
+### dbt_zendesk_source changes (see the [Release Notes](https://github.com/fivetran/dbt_zendesk_source/releases/tag/v0.13.0) for more details)
+- Updated the `stg_zendesk__schedule_holidays` model to allow users to disable holiday processing by setting variable `using_holidays` to `false`.
+- Added field-level documentation for the `stg_zendesk__audit_log` table.
+
+## Bug Fixes
+- Resolved a bug in the `int_zendesk__schedule_spine` model where users experienced large gaps in non-holiday periods. The updated logic addresses this issue.
 
+## Decision log
+- Added the following [DECISIONLOG](https://github.com/fivetran/dbt_zendesk/blob/main/DECISIONLOG.md) entries:
+  - Entry addressing how multiple schedule changes in a single day are handled. Only the last change of the day is captured to align with day-based downstream logic.
+  - Entry to clarify backfilling of schedule history. The most recent schedule is sourced from `stg_zendesk__schedule`, while historical changes are managed separately, allowing users to disable the history feature if needed.
+
+## Under the Hood
+- Replaced instances of `dbt.date_trunc` with `dbt_date.week_start` to standardize week start dates to Sunday across all warehouses, since our schedule logic relies on consistent weeks.
+- Replaced the deprecated `dbt.current_timestamp_backcompat()` function with `dbt.current_timestamp()` to ensure all timestamps are captured in UTC.
```

Comment:
Should we mention which models folks (esp those on Snowflake) may see differences in as a result?

Reply:
Hmmm I'm leaning towards no since I'm not even sure. It was mainly precautionary.

Reply:
Fair!